src/main/java/frc/robot/swerve/SwerveDrive.java

```diff
@@ -188,15 +184,11 @@ public boolean encodersConnected() {
         return connected;
     }
 
-    public Command checkModule(int moduleIndex){
-        return new RunCommand(
-                ()-> modules[moduleIndex].checkModule(), this);
-    }
-
-    public void checkSwerve() {
-        for (int i = 0; i < 4; i++) {
-            checkModule(i);
-        }
+    public Command checkSwerve() {
+        var command = Arrays.stream(modules).map((module)-> run(module::checkModule))
+                .reduce(Commands.none(), Commands::parallel);
```

Comment:
I'd change the import such that's it'll be just `none()` and `parallel` (not sure though if the second one is possible).
Also, use `setName()`. The reasoning for that is explained here: https://discord.com/channels/176186766946992128/368993897495527424/1188615067768467476
Anyway, great usage of Java stream API and the new commands paradigm (I know it's ChatGPT, but still).

Reply:
> I'd change the import such that's it'll be just none() and parallel

It's not possible.

>The reasoning for that is explained here

I'm probably not in that server so the link doesn't work.

>I know it's ChatGPT, but still

It's not chatGPT it's eitanGPT. It's a very impressive AI.

Reply:
1. Wouldn't a star import make this possible?
2. Here's the explanation:
  > If you're wondering what the comment is talking about, if you use the old subclassing approach to defining commands, it will use the class's name as the command name, which is *usually* good enough. However, with inlining (the preferred approach), the class name would be `ParallelCommandGroup` or something similar (as Asa said), which would make it hard to identify, so using `withName()` is encouraged to give it a meaningful name.
3. Ask EitanGPT where it's got the code from 😉

Also, just noticed a possible bug – `run(module::checkModule)` might be problematic because you put multiple commands with the same subsystem required. This might behave weirdly. In this case you should use the requirements-less `Commands.run()` then add the retirement to the composed command as it's already done.

One last thing – you should use `withName()` instead of `setName()` because the former returns the command, so you can chain it to the commands creation.

Reply:
I still find it insulting that you assume I didn't write the code...

Reply:
My apologies. This comment was written before Friday, I didn't know then it's not yours :-)

Reply:
I'm already importing `edu.wpi.first.wpilibj2.command.Commands`